subprojects/dependency-management/src/test/groovy/org/gradle/api/internal/artifacts/transform/TransformationNodeSpec.groovy

```diff
@@ -72,13 +72,13 @@ class TransformationNodeSpec extends Specification {
         node.resolveDependencies(dependencyResolver, hardSuccessor)
 
         then:
-        1 * hardSuccessor.execute(initialNode)
-        1 * graphDependenciesResolver.computeDependencyNodes(transformationStep) >> container
-        1 * dependencyResolver.resolveDependenciesFor(null, container) >> [additionalNode]
-        1 * hardSuccessor.execute(additionalNode)
         1 * transformationStep.dependencies >> transformerDependencies
         1 * dependencyResolver.resolveDependenciesFor(null, transformerDependencies) >> [isolationNode]
         1 * hardSuccessor.execute(isolationNode)
+        1 * graphDependenciesResolver.computeDependencyNodes(transformationStep) >> container
```

Comment:
order matters now (Thanks @bigdaz)

Reply:
Is this documented somewhere?

Reply:
Unfortunately I couldn't find it, I'm not even sure what Groovy has to do here, I thought this was Spock magic...